specification/dataprotection/resource-manager/readme.ruby.md

```diff
@@ -13,6 +13,17 @@ azure-arm: true
 ``` yaml $(ruby) && $(multiapi)
 batch:
   - tag: package-2021-02-preview
+  - tag: package-2021-01
+```
+
+### Tag: package-2021-01 and ruby
```

Comment:
Please dont update ruby. We  will deprecate ruby sdk.

Reply:
Removed package-2021-01 for Ruby